src/props/onAuth.js

```diff
@@ -19,31 +18,38 @@ export type OnAuth = (params : XOnAuthDataType) => ZalgoPromise<string | void>;
 type GetOnAuthOptions = {|
     facilitatorAccessToken : string,
     createOrder : CreateOrder,
-    upgradeLSAT : boolean,
-    userIDToken : ?string,
     clientID : string
 |};
 
-export function getOnAuth({ facilitatorAccessToken, createOrder, upgradeLSAT, userIDToken, clientID } : GetOnAuthOptions) : OnAuth {
-    upgradeLSAT = (upgradeLSAT || Boolean(userIDToken) || upgradeLSATExperiment.isEnabled()) && LSAT_UPGRADE_EXCLUDED_MERCHANTS.indexOf(clientID) === -1;
+export function getOnAuth({ facilitatorAccessToken, createOrder, clientID } : GetOnAuthOptions) : OnAuth {
+    const upgradeLSAT = LSAT_UPGRADE_EXCLUDED_MERCHANTS.indexOf(clientID) === -1;
 
     return ({ accessToken } : XOnAuthDataType) => {
         getLogger().info(`spb_onauth_access_token_${ accessToken ? 'present' : 'not_present' }`);
 
         return ZalgoPromise.try(() => {
             if (accessToken) {
-                upgradeLSATExperiment.logStart();
-
                 if (upgradeLSAT) {
                     return createOrder()
-                        .then(orderID => upgradeFacilitatorAccessToken(facilitatorAccessToken, { buyerAccessToken: accessToken, orderID }))
-                        .then(() => {
-                            getLogger().info('upgrade_lsat_success');
+                        .then(orderID => {
+                            if (window.xprops.createSubscription) {
+                                return null;
+                            }
+
+                            return upgradeFacilitatorAccessToken(facilitatorAccessToken, { buyerAccessToken: accessToken, orderID });
+                        })
+                        .then(data => {
+                            getLogger().info(`upgrade_lsat_success_${ data ? JSON.stringify(data) : 'null' }`);
+
+                            if (!data || (data && !data.upgradeLowScopeAccessToken)) {
```

Comment:
You could use optional chaining here:
```suggestion
                            if (!data || !data?.upgradeLowScopeAccessToken) {
```